fastchat/conversation.py

```diff
@@ -469,6 +469,20 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# Snoozy default template
+# Reference: https://github.com/nomic-ai/gpt4all/blob/d4861030b778da6db59d21d2927a4aba4f9f1f43/gpt4all-bindings/python/gpt4all/gpt4all.py#L232
+register_conv_template(
+    Conversation(
+        name="snoozy",
+        system="The prompt below is a question to answer, a task to complete, or a conversation to respond to; decide which and write an appropriate response.\n",
+        roles=("user", "assistant"),
```

Comment:
Please try to strictly follow https://github.com/nomic-ai/gpt4all/blob/d4861030b778da6db59d21d2927a4aba4f9f1f43/gpt4all-bindings/python/gpt4all/gpt4all.py#L232.

For example, they use "### Instruction"/"### Response" instead of "user" / "assistant"